components/Breadcrumb/src/index.scss

```diff
@@ -1,18 +1,32 @@
 .denhaag-breadcrumb {
-  --denhaag-breadcrumb-background-color: transparent;
+  --denhaag-breadcrumb-link-background-color: transparent;
 
+  background-color: var(--denhaag-breadcrumb-background-color, transparent);
   color: var(--denhaag-breadcrumb-color, inherit);
+  padding-block-end: var(--denhaag-breadcrumb-padding-block);
+  padding-block-start: var(--denhaag-breadcrumb-padding-block);
+  padding-inline-end: var(--denhaag-breadcrumb-padding-inline);
+  padding-inline-start: var(--denhaag-breadcrumb-padding-inline);
+}
+
+@media (min-width: 1124px) {
+  .denhaag-breadcrumb {
+    --denhaag-breadcrumb-padding-inline: 0;
+  }
 }
 
 .denhaag-breadcrumb__list {
   block-size: 100%;
   display: flex;
+  margin-block-end: 0;
+  margin-block-start: 0;
+  margin-inline-end: auto;
+  margin-inline-start: auto;
+  max-width: var(--denhaag-breadcrumb-max-width, 100%);
```

Comment:
If this `--denhaag-breadcrumb-max-width` is only used for the `__list` modifier, than the token can be named `--denhaag-breadcrumb-list-max-width`

Reply:
Edited to `--denhaag-breadcrumb-list-max-width`